extensions/github-gist/src/util/utils.ts

```diff
@@ -2,13 +2,14 @@ import { parse } from "path";
 import { Alert, confirmAlert, Icon } from "@raycast/api";
 import ActionStyle = Alert.ActionStyle;
 import { GistItem } from "./gist-utils";
+import { encodeURI } from "js-base64";
 
 export const isEmpty = (string: string | null | undefined) => {
   return !(string != null && String(string).length > 0);
 };
 
 export const raySo = (title: string, content: string) =>
-  `https://ray.so/?colors=cnady&background=true&darkMode=false&padding=32&title=${title}&code=${content}&language=auto`;
+  `https://ray.so/#colors=cnady&background=true&darkMode=false&padding=32&title=${title}&code=${encodeURI(content)}&language=auto`;
```

Comment:
Thought (non-blocking): It would be great if we can provide a preference for theme to use. The current themes are: https://ray.so/api/config

Reply:
I suspect that reviewers will review much faster if the PR is singly-focused.

It is a small PR with only two changes; however, currently one change is security/attack-surface focused; while the other is an unrelated utility change.

If the PR is focused to a single thing, I believe it will get reviewed much faster.

Reply:
+1 that's why I suggested it is non-blocking. Just a suggestion for future iteration.

Reply:
@jfkisafk I see what you're saying; I think it's non-ideal to increase the cognitive load of this (or any) PR with anything (even tiny) outside of the original scope (no pun intended) of the PR.

It might seem nitpicky; however, it isn't an ideal practice and is a bit of an anti-pattern.